Meta doesn't start on master

## This is a...

<pre><code>
[ ] Feature request
[X] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description
```
org.springframework.beans.factory.BeanDefinitionStoreException: Failed to process import candidates for configuration class [io.syndesis.connector.meta.Application]; nested exception is java.lang.IllegalStateException: Unable to read meta-data for class io.syndesis.integration.runtime.sb.SyndesisHttpConfiguration
	at org.springframework.context.annotation.ConfigurationClassParser.processDeferredImportSelectors(ConfigurationClassParser.java:561) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassParser.parse(ConfigurationClassParser.java:187) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.processConfigBeanDefinitions(ConfigurationClassPostProcessor.java:308) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.postProcessBeanDefinitionRegistry(ConfigurationClassPostProcessor.java:228) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:272) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:92) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:687) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:525) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at io.syndesis.connector.meta.Application.main(Application.java:47) [classes!/:1.7-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_151]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_151]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_151]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_151]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) [meta-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) [meta-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) [meta-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:595) [meta-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
Caused by: java.lang.IllegalStateException: Unable to read meta-data for class io.syndesis.integration.runtime.sb.SyndesisHttpConfiguration
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.getAnnotationMetadata(AutoConfigurationSorter.java:217) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.getAnnotationValue(AutoConfigurationSorter.java:198) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.readBefore(AutoConfigurationSorter.java:186) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.<init>(AutoConfigurationSorter.java:158) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClasses.<init>(AutoConfigurationSorter.java:115) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter.getInPriorityOrder(AutoConfigurationSorter.java:57) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationImportSelector.sort(AutoConfigurationImportSelector.java:241) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationImportSelector.selectImports(AutoConfigurationImportSelector.java:98) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassParser.processDeferredImportSelectors(ConfigurationClassParser.java:552) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	... 22 common frames omitted
Caused by: java.io.FileNotFoundException: class path resource [io/syndesis/integration/runtime/sb/SyndesisHttpConfiguration.class] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172) ~[spring-core-4.3.18.RELEASE.jar!/:4.3.18.RELEASE]
	at org.springframework.core.type.classreading.SimpleMetadataReader.<init>(SimpleMetadataReader.java:50) ~[spring-core-4.3.18.RELEASE.jar!/:4.3.18.RELEASE]
	at org.springframework.core.type.classreading.SimpleMetadataReaderFactory.getMetadataReader(SimpleMetadataReaderFactory.java:102) ~[spring-core-4.3.18.RELEASE.jar!/:4.3.18.RELEASE]
	at org.springframework.boot.type.classreading.ConcurrentReferenceCachingMetadataReaderFactory.createMetadataReader(ConcurrentReferenceCachingMetadataReaderFactory.java:89) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.type.classreading.ConcurrentReferenceCachingMetadataReaderFactory.getMetadataReader(ConcurrentReferenceCachingMetadataReaderFactory.java:76) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.core.type.classreading.SimpleMetadataReaderFactory.getMetadataReader(SimpleMetadataReaderFactory.java:80) ~[spring-core-4.3.18.RELEASE.jar!/:4.3.18.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.getAnnotationMetadata(AutoConfigurationSorter.java:213) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	... 30 common frames omitted
```

--- a/app/connector/http/src/main/resources/META-INF/spring.factories
+++ b/app/connector/http/src/main/resources/META-INF/spring.factories
@@ -1,3 +1,2 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-  io.syndesis.connector.http.HttpVerifierAutoConfiguration,\
-  io.syndesis.integration.runtime.sb.SyndesisHttpConfiguration
+  io.syndesis.connector.http.HttpVerifierAutoConfiguration

--- a/app/connector/webhook/src/main/resources/META-INF/spring.factories
+++ b/app/connector/webhook/src/main/resources/META-INF/spring.factories
@@ -1,3 +1,2 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-  io.syndesis.connector.webhook.WebhookServletAutoConfiguration,\
-  io.syndesis.integration.runtime.sb.SyndesisHttpConfiguration
+  io.syndesis.connector.webhook.WebhookServletAutoConfiguration

--- a/app/meta/src/main/resources/application.yml
+++ b/app/meta/src/main/resources/application.yml
@@ -39,13 +39,12 @@ endpoints:
     sensitive: false
     enabled: true
 
-# Disable ActiveMQ auto-configuration of an embedded Broker
+# disable non needed auto-configuration
 spring:
   autoconfigure:
     exclude:
       - org.springframework.boot.autoconfigure.jms.activemq.ActiveMQAutoConfiguration
       - org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
-
 io:
   syndesis:
     connector:

